src/plugin/suncalc/lightstrip.js

```diff
@@ -84,7 +101,9 @@ plugin.suncalc.LightStripCtrl.prototype.destroy_ = function() {
   this.updateDelay_ = null;
 
   if (this.element_ && this.resizeFn_) {
-    this.element_.parent().removeResize(this.resizeFn_);
+    if (typeof this.element_.parent().removeResize === 'function') {
```

Comment:
Please remove this, and instead add `jquery.resize.js` to the Karma configuration.